app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from typing import List
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: List[Player]) -> int:
```

Comment:
In the newest python versions you can use default list instead of typing.List
I believe, you can do it in your case